integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -243,7 +243,7 @@ case class CarbonLoadDataCommand(
       } catch {
         case CausedBy(ex: NoRetryException) =>
           // update the load entry in table status file for changing the status to marked for delete
-          if (isUpdateTableStatusRequired) {
+          if (isUpdateTableStatusRequired && !table.isHivePartitionTable) {
```

Comment:
This check is not required because isUpdateTableStatusRequired  flag will be set to true only if the table is not a partition table

Reply:
ok